.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+    # branches:
+    # - master
+  pull_request:
+  # Run daily at 0:01 UTC
+  schedule:
+  - cron:  '1 0 * * *'
+  workflow_dispatch:
```

Comment:
I think this is supposed to enable a button to run the workflow manually, but I'm not sure why we'd need this on top of the other rules (also I don't know where the button is supposed to be found)

Reply:
I would highly suggest that you keep it. It is useful.

Reply:
It doesn't seem like it actually costs anything so why not.